xknx/devices/climate.py

```diff
@@ -164,63 +208,74 @@ def is_on(self):
     @property
     def initialized_for_setpoint_shift_calculations(self):
         """Test if object is initialized for setpoint shift calculations."""
-        if not self.setpoint_shift.initialized:
+        if not self._setpoint_shift.initialized:
             return False
-        if self.setpoint_shift.value is None:
+        if self.setpoint_shift is None:
             return False
         if not self.target_temperature.initialized:
             return False
         if self.target_temperature.value is None:
             return False
         return True
 
-    async def set_target_temperature(self, target_temperature):
-        """Calculate setpoint shift shift and send it to  KNX bus."""
+    @property
+    def temperature_step(self):
+        """Return smallest possible temperature step."""
         if self.initialized_for_setpoint_shift_calculations:
-            await self.set_target_temperature_setpoint_shift(target_temperature)
-        # broadcast new target temperature and set internally
-        await self.target_temperature.set(target_temperature)
+            return self._setpoint_shift.setpoint_shift_step
+        return 0.1
```

Comment:
If we returned None (for temperature_address actors) HA would default to a 0.5 resolution. This could behave awkward if other devices on the KNX bus support 0.1 resolution.
We could either 
- return 0.1 per default (not really elegant)
- check for None in the HA plugin and return 0.1 there (even less elegant) or 
- change `setpoint_shift_step` to something like `temperature_resolution` and use it for temperature_address actors and setpoint_shift actors alike

Reply:
it returns DEFAULT_TEMPERATURE_STEP now. Are you ok with this or shall we move this to the HA component?

Reply:
For me `DEFAULT_TEMPERATURE_STEP` would be enough, but lets wait for @Julius2342